### ISSUE TITLE
Lazybones no longer available on SDKMAN

### ISSUE THREAD

**Issue by cbmarcum** [griffon/griffon#312]
I've discovered since the Bintray shutdown that Lazybones is no longer a candidate on SDKMAN.
I've forked Lazybones [1] so I could fix the builds and get the lazybones app working for other projects of mine.
I contacted SDKMAN and someone needs to make all the previous releases available to continue Lazybones on SDKMAN or rebrand it and move forward.
Do you have interest in continuing to use the Lazybones tool or a successor for templates going forward if I get it picked up again?
Thanks,
Carl
[1] https://github.com/cbmarcum/lazybones

**Comment by aalmiray** [comment]
Yes, that'd be great! I miss Lazybones. Be aware that Gradle 5.1 broke Lazybones, they changed how the Zip task works internally.

The Griffon build will remain in Gradle 6.9 for quite some time given how Gradle 7 has broken its plugin ecosystem. It'll take a few more months for the dust to settle and most plugins working properly with Gradle 7.x.
